docs/ENV.md

```diff
@@ -269,19 +269,19 @@ Additionally, you can search for the following:
 ### `POSTFIX_RELAYHOST`
 - **Explaination:** Host that relays your mails.
 - **Valid Values:** IP address | Domain
-- **Default:** --BLANK--
-- **Example:** smtp.gmail.com:587
+- **Default:** False
```

Comment:
As disused, using False for an ENV that doesn't require only True|False seems confusing.  Recommend using null or None since defaults values are required by Docker and can not be left blank.

Reply:
Done! :smile: